lib/context_adapter_logger.js

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2015 Telefónica Investigación y Desarrollo, S.A.U
+ *
+ * This file is part of context-adapter
+ *
+ * context-adapter is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * context-adapter is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public
+ * License along with context-adapter.
+ * If not, seehttp://www.gnu.org/licenses/.
+ *
+ * For those usages not covered by the GNU Affero General Public License
+ * please contact with::[german.torodelvalle@telefonica.com]
+ */
+
+(function() {
+  'use strict';
+
+  var caConfig = require('./context_adapter_configuration');
+  var fs = require('fs');
+  var mkdirp = require('mkdirp');
+  var path = require('path');
+  var winston = require('winston');
```

Comment:
Logops, no doubt about it. There is an issue already in most of the repositories that are using that old version to change it to logops. That repository was a previous version o the same thing we kept for some time until the official logops repository was released in Github..